### DISCUSSION TITLE
大佬可以适配无线宝雅典娜吗（JDCloud RE-CS-02）

### DISCUSSION THREAD

**Discussion by aYangQT** [immortalwrt/immortalwrt#1438] [General]
如题，刚买跑了半个月宽带被限制了，不能退货了，想刷immortalwrt用

**Comment by breeze303** [comment]
![image](https://github.com/user-attachments/assets/4126be41-9f42-4d12-a6c9-70f0466d7f3b)
https://github.com/King-Of-Knights/openwrt-6.x
雅典娜的话你可以尝试下这个库，imm目前貌似是不会特地添加对高通机器的支持只是同步主线op的支持，那个库的是基于op主线添加了高通开源无线及nss支持，后面移植了imm的功能进去，你可以尝试下

**Reply by aYangQT** [reply]
好的，有时间研究下，我的固件更新了，不知道怎么开启ssh了，估计要拆机刷了
